exchange/utils.go

```diff
@@ -75,13 +66,20 @@ func cleanOpenRTBRequests(ctx context.Context,
 		LMT:   lmtPolicy.ShouldEnforce(),
 	}
 
+	privacyLabels.CCPAProvided = ccpaPolicy.Value != ""
+	privacyLabels.CCPAEnforced = privacyEnforcement.CCPA
+	privacyLabels.COPPAEnforced = privacyEnforcement.COPPA
+	privacyLabels.LMTEnforced = privacyEnforcement.LMT
+
 	if gdpr == 1 {
-		cleanMetrics.gdprEnforced = true
+		privacyLabels.GDPREnforced = true
 		parsedConsent, err := vendorconsent.ParseString(consent)
 		if err == nil {
-			cleanMetrics.gdprTcfVersion = int(parsedConsent.Version())
+			version := int(parsedConsent.Version())
+			privacyLabels.GDPRTCFVersion = pbsmetrics.TCFVersionValue(version)
```

Comment:
Shouldn't this call refer to the `TCFVersionToValue` function found in `pbsmetrics/metrics.go` instead?
```
280 func TCFVersionToValue(version int) TCFVersionValue {
281     switch {
282     case version == 1:
283         return TCFVersionV1
284     case version == 2:
285         return TCFVersionV2
286     }
287     return TCFVersionErr
288 }
pbsmetrics/metrics.go
```

Reply:
Despite the fact that this part of the code appears to be covered by test cases, I could not find a test inside the `exchange` directory asserting the contents of the third return value of `cleanOpenRTBRequests`, `privacyLabels pbsmetrics.PrivacyLabels`. Should we include a test to assert this logic?

![image](https://user-images.githubusercontent.com/15863840/86846449-df1c3e00-c078-11ea-8228-e1985b7cbd18.png)

Reply:
You're exactly right. Bug fixed. Tests added.